hazelcast/src/main/java/com/hazelcast/internal/nio/DefaultAggregateEndpointManager.java

```diff
@@ -79,4 +84,16 @@ public void addConnectionListener(ConnectionListener listener) {
             manager.addConnectionListener(listener);
         }
     }
+
+    @Override
+    public Map<EndpointQualifier, NetworkStats> getNetworkStats() {
+        Map<EndpointQualifier, NetworkStats> stats = null;
+        for (Map.Entry<EndpointQualifier, EndpointManager<TcpIpConnection>> entry : endpointManagers.entrySet()) {
+            if (stats == null) {
+                stats = new HashMap<>();
```

Comment:
emptyMap()

Reply:
Are you sure? This map is filled in with data later on. `emptyMap()` returns a singleton, which won't work in this case.

Reply:
Sorry my bad, I neglected the surrounded code.